README.md

```diff
@@ -22,36 +22,65 @@ A Very Good Infinite List Widget created by [Very Good Ventures][very_good_ventu
 
 ## Usage
 
-A basic `InfiniteList` requires two parameters:
+The `InfiniteList` API is very similar to that of `ListView.builder`. A basic implementation requires four parameters:
 
-- `itemLoader` which is responsible for asynchronously fetching the content
-- `builder` which is responsible for returning a `Widget` given a specific item (result)
+- An `itemCount` that represents the amount of items that should be rendered using the `itemBuilder`.
+- An `itemBuilder` that is responsible for returning a widget for every index of the `itemCount`.
+- An `hasReachedMax` flag that indicates if any more data is available.
```

Comment:
I would propose to make this property have a default value of false. When you pass false and use it with bloc it works just fine. Might be something there that I am missing but even examples show passing false as parameter and do not check any condition;

Reply:
Great point! The original intention was to have the user explicitly know and tell us that the list has reached the final value or not (in the same way that `TextButton.onPressed` is `required` but also nullable). I'll make the change so that it default to false 😄